oecophylla/distance/sourmash.rule

```diff
@@ -0,0 +1,60 @@
+rule sourmash_sig:
+    """
+    Create a minhash signature for the input forward and reverse fastq read files of every sample.
+    """
+    input:
+        forward = qc_dir + "{sample}/filtered/{sample}.R1.trimmed.filtered.fastq.gz",
+        reverse = qc_dir + "{sample}/filtered/{sample}.R2.trimmed.filtered.fastq.gz"
+    output:
+        sketch = distance_dir + '{sample}/sourmash/{sample}.sig'
+    params:
+        env = config['envs']['distance'],
+        scaled = config['params']['sourmash']['scaled'],
+        kmer = config['params']['sourmash']['kmer']
+    threads:
+        1
+    log:
+        distance_dir + "logs/sourmash_sig.sample=[{sample}].log"
+    benchmark:
+        "benchmarks/distance/sourmash_sig.sample=[{sample}].txt"
+    run:
+        with tempfile.TemporaryDirectory(dir=TMP_DIR_ROOT) as temp_dir:
+            shell("""
+                  set +u; {params.env}; set -u
+                  zcat {input.forward} {input.reverse} > {temp_dir}/{wildcards.sample}
+                  sourmash compute --scaled {params.scaled} -k {params.kmer} -o {output[0]}
```

Comment:
`-o` should be `{output}`, not `{output[0]}`

Reply:
better to be explicit?